dmriprep/interfaces/reports.py

```diff
@@ -30,6 +30,15 @@
 \t</ul>
 """
 
+DIFFUSION_TEMPLATE = """\
+\t\t<details open>
+\t\t<summary>Summary</summary>
+\t\t<ul class="elem-desc">
+\t\t\t<li>Distinct shells: {shells_dist}</li>
```

Comment:
- Why don't we also print out the shell distribution?
- What happens with DSI schemes? Maybe we can add a unit test of the `DiffusionSummary` interface?

Reply:
Right now, `shells_dist` does print out the shell distribution. But perhaps I should give it a different label and variable name to make this more clear.

I've tested this on the `taiwan_ntu_dsi` dataset that comes bundled with `dipy`. Here's what it outputs:
```
{0: 7, 100: 12, 200: 8, 400: 6, 600: 24, 900: 24, 1500: 12, 1900: 30, 2400: 24, 2900: 24, 3400: 8, 4000: 24}
```
I haven't worked with DSI data before so I'm not quite sure if this is what we're looking for or whether this should be improved.

Will update with a unit test for the new interface!